src/import-data/ImportData.ts

```diff
@@ -75,7 +75,7 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
         jobId,
         wdsProxyUrl: wdsUrl,
       };
-      asyncImportJobStore.update(Utils.append(newJob));
+      asyncImportJobStore.update(Utils.append<AsyncImportJob>(newJob));
```

Comment:
Would you consider doing a direct import of `append` here like you did in index.ts to lower the friction of removing the `*` import?